teamsupport/models.py

```diff
@@ -62,3 +87,28 @@ def __init__(self, client, user_id=None, data=None):
                 "__init__() needs either a 'user_id' or 'data' argument "
                 '(neither given)')
         self.id = self.UserID
+
+
+class Contact(XmlModel):
+    def __init__(self, client, contact_id=None, data=None):
+        self.client = client
+        self.data = data
+        if contact_id:
+            self.data = client.get_contact(contact_id)
+        elif not self.data:
+            raise MissingArgumentError(
+                "__init__() needs either a 'contact_id' or 'data' argument "
+                '(neither given)')
+        self.id = self.ContactID
```

Comment:
I don't think we need all that logic from Ticket model here.  I would say accepting `data` in constructor would be enough for us.  Besides it looks like we don't have `get_contact` method